scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -0,0 +1,971 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" A Script to process USA Census PEP monthly population data
+    from the datasets in provided local path.
+    Typical usage:
+    1. python3 preprocess.py
+    2. python3 preprocess.py -i input_data
+"""
+import os
+import re
+import sys
+# For import util.alpha2_to_dcid
```

Comment:
Pls remove unused code.

Reply:
This is a comment stating that alpha2_to_dcid is being imported from util for geoId. I made the comment more specific.